packages/mds-stream/kafka/stream-consumer.ts

```diff
@@ -108,6 +108,11 @@ const createStreamConsumer = async (
   { clientId = 'client', groupId = 'group', fromBeginning = false }: Partial<KafkaStreamConsumerOptions> = {}
 ) => {
   try {
+    const componentId = 'kafka-consumer-pre-connection'
```

Comment:
Mind calling this `preConnectionComponentId` or something?

Reply:
Sure